tests/data/NotoSans-MM-ASCII-VF.ttx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ttFont sfntVersion="\x00\x01\x00\x00" ttLibVersion="4.4">
```

Comment:
(not important: did the ttLib version go backwards?)

Reply:
I changed this because requirements.txt is still pinned to 3.44 and tox installs that in venvs. Not sure why that wasn't caught before. Will update requirements in a separate PR.